examples/deno.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Bot,
+  Context,
+  session,
+  SessionFlavor,
+} from "https://deno.land/x/grammy@v1.9.0/mod.ts";
+import { I18n, I18nFlavor } from "../src/mod.ts";
+
+interface SessionData {
```

Comment:
Maybe we should have an interface for I18n so `SessionData` can extends from it. It will help other devs understand what Session Data has. It's also better for type checking as well.

Reply:
Can you explain a little bit more? I didn't get the idea.

Reply:
I mean like this:

```ts
// Plugin
interface I18nSession {
   __language_code?: string;
}


// bot.ts
interface SessionData extends I18nSession {
    apples: number;
}
```

This way will help us easily extend plugin later also other dev don't have to remember what properties to add.

Reply:
I don't personally think this is necessary. Users only have to add `__language_code` to their session data type if they want edit it by themselves in the rest of their code. No matter if they provide it in the session data or not, if `useSession` is set to true, `__language_code` is set by the plugin internally. No one has to worry about anything. Not even another developer, because `__language_code` is used at very few parts of the code.